packages/test/package.json

```diff
@@ -14,26 +14,30 @@
   "repository": {
     "type": "git",
     "url": "git+https://github.com/theforeman/foreman-js.git",
-    "directory": "packages/env"
+    "directory": "packages/test"
   },
   "scripts": {
     "test": "echo \"@theforeman/env does not contain any tests at the moment.\""
   },
   "bin": {
-    "tfm-test": "./bin/tfm-test.js"
+    "tfm-test": "./bin/tfm-test.js",
+    "tfm-coveralls": "./bin/tfm-coveralls.js"
   },
   "dependencies": {
     "@theforeman/builder": "^3.12.0",
     "@theforeman/vendor": "^3.12.0",
```

Comment:
Are you sure you need `@theforeman/vendor`? looks like you don't use it.

Reply:
nice catch